Gemfile

```diff
@@ -7,6 +7,7 @@ platforms :rbx do
   gem 'rubinius-developer_tools'
 end
 
-gem 'country_select', '~> 2.5.2'
-gem 'railties'
+gem 'country_select', '~> 3.1.1'
+gem 'activemodel', '~> 5.2.0.rc2'
+gem 'railties', '~> 5.2.0.rc2'
```

Comment:
Now that rails 5.2 has been released, you should update these.

Reply:
Done :)